public/components/notebooks/components/notebook.tsx

```diff
@@ -157,15 +157,14 @@ export class Notebook extends Component<NotebookProps, NotebookState> {
         'Error parsing paragraphs, please make sure you have the correct permission.',
         'danger'
       );
-      console.error(err);
```

Comment:
i thought eslint disabled no-console? https://github.com/opensearch-project/dashboards-observability/blob/8de4b3e5577a7345c64ac5dfeab99f080f28578b/.eslintrc.js#L17

Reply:
It's not working then...

```
/Users/pjfitz/IdeaProjects/OpenSearch/dashboards/plugins/dashboards-observability/public/components/notebooks/components/notebook.tsx
  160:7   error  Unexpected console statement                   no-console
  216:11  error  Unexpected console statement                   no-console
  258:15  error  Unexpected console statement                   no-console
  363:9   error  Unexpected console statement                   no-console
```